Add Repeat to the IFixture interface issue #540

SeanFarrow opened the pull request:
Reopening the pr as I inadvertently closed it!

ploeh commented on Src/AutoFixtureUnitTest/FixtureTest.cs:
```diff
@@ -1847,12 +1847,11 @@ public void DefaultRepeatCountIsThree()
         {
             // Fixture setup
-            Fixture sut = new Fixture();
+            IFixture sut = new Fixture();
```
Why was this test changed to work on `IFixture`? Doesn't it work on `Fixture` any longer?

ploeh commented:
Looking good :+1: 

It looks like there are a few issues to address, but they look minimal to me.

SeanFarrow commented:
I think there all fixed, I’ve reverted the tests to use the Fixture class and fixed all the documentation warnings.

Cheers

Sean.

 

From: Mark Seemann [mailto:notifications@github.com] 
Sent: 14 March 2016 16:46
To: AutoFixture/AutoFixture <AutoFixture@noreply.github.com>
Cc: Sean Farrow <s.r.farrow@icloud.com>
Subject: Re: [AutoFixture] Add Repeat to the IFixture interface issue #540 (#564)

 

Looking good   <https://assets-cdn.github.com/images/icons/emoji/unicode/1f44d.png> 

It looks like there are a few issues to address, but they look minimal to me.

—
Reply to this email directly or view it on GitHub <https://github.com/AutoFixture/AutoFixture/pull/564#issuecomment-196405771> .  <https://github.com/notifications/beacon/ABY1flGmz5ashS0GTQiqnrd_G6Z3eYP9ks5ptZDcgaJpZM4HwHc9.gif>

ploeh commented:
Almost there, but the `FixtureTest` code looks funny, now.

Basically, I'd expect that that file wasn't touched at all...

ploeh commented:
Thank you for your contribution! It's now live as [AutoFixture 3.43.2](https://www.nuget.org/packages/AutoFixture/3.43.2).
